app/controllers/settings.controller.js

```diff
@@ -0,0 +1,95 @@
+const { where } = require("sequelize");
+const db = require("../models");
+const Op = db.Sequelize.Op;
+const Setting = db.settings;
+
+// Insert a new setting into the database
+exports.create = (req, res) => {
+    if (req.body.name === undefined) {
+        const error = new Error("Name for setting can't be empty!");
+        error.statusCode = 400;
+        throw error;
+    }
+
+    const setting = {
+        name: req.body.name
+    };
+
+    Setting.create(setting)
+        .then((data) => {
+            res.send(data);
+        })
+        .catch((err) => {
+          res.status(404).send({
```

Comment:
I think most of these 404 status codes are not appropriate.  404 means "not found", i.e., you were trying to do something and the resource or object you referenced doesn't exist.  In this case here, for example, you are trying to create a setting in the backend DB and it fails with an error, it is very unlikely that anything about this failure is due to something "not being found".  It's more likely the DB is down or inaccessible, or the setting(s) data you are trying to insert is invalid in some way (maybe it violates a length constraint or has special characters in it that can't be inserted into the DB as-is).  In any event, I think many of these uses of `.status(404)` are better off as HTTP 500 or similar